src/main/java/io/swagger/oas/inflector/config/DefaultControllerFactory.java

```diff
@@ -14,9 +14,10 @@
  *  limitations under the License.
  */
 
-package io.swagger.inflector.config;
+package io.swagger.oas.inflector.config;
 
-import io.swagger.models.Operation;
+
+import io.swagger.oas.models.Operation;
 
 /**
  * Default ControllerFactory implementation that just calls newInstance
```

Comment:
what is the purpose of this class? i'm not sure if we should invoke a one line method creating an instance of this class.